app/forms/steps/capital/national_savings_certificates_form.rb

```diff
@@ -5,13 +5,17 @@ class NationalSavingsCertificatesForm < Steps::BaseFormObject
       include OwnershipConfirmation
       include ApplicantOrPartner
 
+      # matches any alphanumeric string that does not end with '-' to avoid modsec errors
+      NSC_REGEXP = /.*(?<!-)\z/
```

Comment:
@RoseSAK Should we only invalidate '-' or any other special character?. 
If we know format of 'holder_number'  then we can validate according to the format. May be allow only characters and numbers only.

Reply:
Hmm yes good point, I will check with Chris what the format would likely be